enforcer/src/main/java/org/wso2/micro/gateway/enforcer/config/ConfigHolder.java

```diff
@@ -103,33 +113,23 @@ private void populateJWTIssuerConfiguration() throws KeyStoreException {
             issuerDto.setJwksConfigurationDTO(jwksConfigurationDTO);
 
             String certificateAlias = (String) issuer.get(ConfigConstants.JWT_TOKEN_CERTIFICATE_ALIAS);
-            if (trustStore.getCertificate(certificateAlias) != null) {
-                Certificate issuerCertificate = trustStore.getCertificate(certificateAlias);
-                issuerDto.setCertificate(issuerCertificate);
+            try {
+                if (trustStore.getCertificate(certificateAlias) != null) {
+                    Certificate issuerCertificate = trustStore.getCertificate(certificateAlias);
+                    issuerDto.setCertificate(issuerCertificate);
+                }
+            } catch (KeyStoreException e) {
+                logger.error("Error while loading certificate with alias " + certificateAlias + " from the trust store",
```

Comment:
any reason for not throwing the error?

Reply:
This is during server startup, if we throw from here that means we will have to kill the server if there is an issue in config reading, we need to decide whether we need to kill the server in case of an failure decide